### ISSUE TITLE
Use HackRF to run Krakensdr

### ISSUE THREAD

**Issue by michalisk13** [greatscottgadgets/hackrf#1470]
### What would you like to know?

Hi,

Do you beleive that it's possible to use hackrf 1 alongside opera cake to perform DF similar to krakensdr but instead of having 5 different receivers to only use a single hackrf receiver which will swiftly switch between 4 or more antennas connected to the opera cake?

I saw that it is possible to do a pseudo-doppler DF, but I was wondering if it's possible to do something similar to kraken.

Thanks,
Mike

**Comment by mossmann** [comment]
No, KrakenSDR uses multiple receivers to simultaneously capture signals from a phased array of antennas. The technique used by KrakenSDR requires simultaneous capture. HackRF One with Opera Cake can capture on only one antenna at a time.

The traditional way to do pseudo-doppler is to use FM. With the antennas arranged in a circle, cycling through the antennas introduces an FM signal with a phase that indicates direction. Other patterns are possible, but I would consider those to be extensions of pseudo-doppler more than being KrakenSDR-like.
